src/server.ts

```diff
@@ -0,0 +1,59 @@
+import type { Server } from 'http';
+import { createServer as createHttpServer } from 'http';
+import { ApolloServer } from 'apollo-server-express';
+import type express from 'express';
+import type { Disposable } from 'graphql-ws';
+import { ApolloServerPluginDrainHttpServer } from 'apollo-server-core';
+import { PORT, isDevelopment } from './env';
+import { createContext } from './context';
+import { schemaWithMiddleware } from './schema';
+
+let serverCleanup: Disposable;
```

Comment:
What is this for? Why did the `createApolloServer` and `startServer` had to change after being moved from `src/index.ts`?

Reply:
this is actually a recommended plugin for server shut down
https://www.apollographql.com/docs/apollo-server/api/plugin/drain-http-server/

Reply:
removed `server.ts` and simplified `index.ts`